src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+
+  margin-right: 16px;
+
+  &--1 :nth-child(-n+1),
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(-n+5) {
```

Comment:
you can use [loop](https://sass-lang.com/documentation/at-rules/control/for) this